app/views/override/credits/credits-advocates.md

```diff
@@ -0,0 +1,27 @@
+# Credits: The Advocates
+  
+[The Advocates](/collections/advocates-advocates/full-program-video) collection was made possible by a grant from the Open Society Foundations (OSF).
+
+<a href="http://www.opensocietyfoundations.org"><img height="60" src="https://s3.amazonaws.com/openvault.wgbh.org/logos/OSF.jpg"
```

Comment:
@caseyedavis12 : Unless there is a compelling reason, I would prefer to minimize the usage of html in md. (Though in this case, along with all the other outbound links, I think we could add a `target="_blank"`, and that would be our compelling reason.)